File: CPP-1/GradeFinder.cpp
#include<iostream>
using namespace std;
float om, tm,p;
int main(){
    cout<<"Welcome to the Grade Calculator"<<endl;
    cout<<"Enter the value of obtained marks: \n";
    cin>>om;
    cout<<"Enter the value of total marks: \n";
    cin>>tm;
    p=om*100/tm;
    cout<<"The percentage is = "<<p<<endl;
    if(p>=95)
    {
        cout<<"Your Grade is A+ ";
    }
    else if((p<95)&&(p>80))
    {
        cout<<"Your Grade is A ";
    }
    else if(p<=50)
    {
        cout<<"Pass";
    }
    return 0;
}

File: Python/Gradefinder.py
a=int(input("Enter the marks obtained in first Subject = "))
b=int(input("Enter the marks obtained in second Subject = "))
c=int(input("Enter the marks obtained in third Subject = "))
d=int(input("Enter the sum of total marks = "))
e=(a+b+c)*100/d
print(e,"%")
if(e>=90):
    print("A grade")
elif(e>=80 and e<=89):
    print("B grade")
elif(e>=70 and e<=79):
    print("C grade")
elif(e>=60 and e<=69):
    print("D grade")
elif(e>=50 and e<=59):
    print("E grade")
elif(e>50 and e<=0):
    print("Fail")
else:
    print("Error!")